packages/model-viewer/src/features/scene-graph/texture-info.ts

```diff
@@ -154,15 +70,9 @@ export class TextureInfo extends ThreeDOMElement implements
 
   setTexture(texture: Texture|null): void {
     const threeTexture: ThreeTexture|null =
-        texture != null ? texture.source[$underlyingTexture] : null;
+        texture != null ? texture.source[$sourceTexture] : null;
     let encoding: TextureEncoding = sRGBEncoding;
     this[$texture] = texture;
-    // Ensures correlatedObjects is up to date.
-    const correlatedObjects = (this[$correlatedObjects] as Set<ThreeTexture>);
```

Comment:
This is an interesting change, I removed the TextureInfo correlatedObject. I think this better for a couple of reasons, 1) the TextureInfo has no real correlation in Three.js, previously the it was correlated to a three-texture, now Texture is correlated to a three-texture 2) this reduces the number of way you can access and change the underlying object and should help reduce changing things from the wrong place.